### DISCUSSION TITLE
SSL HTTPS on synology docker

### DISCUSSION THREAD

**Discussion by ewo68** [pglombardo/PasswordPusher#1984] [General]
Hi, I installed the pglonbardo/pwpush:latest image on a Synology via "container Manager". I am encountering a problem, I really can't get it to work in https. Also I have my own SSL certificate and I don't know how to install it. I searched everywhere in the docs. Without success. I can't even find the configuration files on Synology. Would you have any help to give me?

**Comment by pglombardo** [comment]
Hi @ewo68 - Whats the error?  Where are you in the process?

If it helps, I have a sample nginx configuration [here](https://github.com/pglombardo/PasswordPusher/tree/master/containers/examples/pwpush-and-nginx).  Nginx is used as the SSL termination point that talks to Password Pusher in the backend.

**Comment by ewo68** [comment]
Hello, thank you for your feedback. It's good for me, I succeeded. It was a problem on my side.


Thank you so much.



________________________________
De : Peter Giacomo Lombardo ***@***.***>
Envoyé : mercredi 20 mars 2024 13:52
À : pglombardo/PasswordPusher ***@***.***>
Cc : Emmanuel GUILLIN ***@***.***>; Mention ***@***.***>
Objet : Re: [pglombardo/PasswordPusher] SSL HTTPS on synology docker (Discussion #1984)


Hi @ewo68<https://github.com/ewo68> - Whats the error? Where are you in the process?

If it helps, I have a sample nginx configuration here<https://github.com/pglombardo/PasswordPusher/tree/master/containers/examples/pwpush-and-nginx>. Nginx is used as the SSL termination point that talks to Password Pusher in the backend.

**Reply by pglombardo** [reply]
Great - that's excellent to hear then.  If any other issues/questions come up let me know!
